<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Model_Evaluate_2</class>
 <widget class="QMainWindow" name="Model_Evaluate_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evaluate Window</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Image">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>371</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Pictures/c2p conversion/patrick.jpg</pixmap>
    </property>
   </widget>
   <widget class="QTextEdit" name="Select_Test_Set">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>trainHOG</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Select_Test_set_sign">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Test set</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="Select_Model">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>model.pkl</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Select_Model_sign">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Model</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Model_Import">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Model_Evaluate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluate</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="PRE_E">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Expected</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PRE_T">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>True</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Acc">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>430</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Accuracy</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Image_select">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>380</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="Select">
    <property name="title">
     <string>Select Page</string>
    </property>
   </widget>
   <addaction name="Select"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
